### PR TITLE
fix: remove display of empty discount and access code 

### LINKED ISSUE #6273: Access and Discount Code Page: Record in not unloading.
**Describe the bug**

Record created while creating new discount and access code is not unloading while exiting the page.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to 'Discount page or Access Page'
2. Click on 'create new discount code'
3. come back to discount page without saving form.

**Expected behaviour**

record must unload on exiting.

**Screenshots**

newwww issue

**Additional context**

### PR DESCRIPTION
**Pull request by sachinchauhan2889**
Fixes #6273 

#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/routes/events/view/tickets/access-codes/create.js [hunk 1]
```diff
@@ -12,7 +12,6 @@ export default class CreateRoute extends Route {
   async model() {
     return RSVP.hash({
       accessCode: this.store.createRecord('access-code', {
-        event         : this.modelFor('events.view'),
```
Comment (iamareebjamal) [inline app/routes/events/view/tickets/access-codes/create.js:15]: why?

Comment (iamareebjamal) [reply app/routes/events/view/tickets/access-codes/create.js:15]: ...

Comment (sachinchauhan2889) [reply app/routes/events/view/tickets/access-codes/create.js:15]: https://github.com/fossasia/open-event-frontend/pull/6052 
@iamareebjamal  sir,
Just try to fix it like this. But today i will try roleback attribute on this.

**Comment by sachinchauhan2889** [comment]
closed by mistake

**Pull request closed without merging**